routes/auth/facebook.js

```diff
@@ -0,0 +1,10 @@
+import { Router } from 'express';
+import passport from 'passport';
+const router = Router();
+
+router.get('/', passport.authenticate('facebook', { session: false }), (req, res) => {
+	res.status(200).json('success');
+});
+
```

Comment:
As I remember, there should also be a `/callback` route for social auth
Does it really work without `callback`?

Reply:
Its work.
![image](https://user-images.githubusercontent.com/17512890/42206855-ff6b7baa-7eb0-11e8-833b-062491d6fd38.png)

But I use the callback. Its placed in the myPassport.js
![image](https://user-images.githubusercontent.com/17512890/42206974-62e81616-7eb1-11e8-9120-b2dca433f064.png)

Reply:
I didn't mean the `cb` functon but the `/callback` route
That's what I'm about https://github.com/jaredhanson/passport-facebook#authenticate-requests

Reply:
In my case it is unnecessary. It works perfectly with one route. As I see, callback route need for a redirect.